tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -159,8 +265,37 @@ def test_job_level_scaling(
         multi_instance_types_ice_cr="ice-cr-multiple",
     )
 
-    _submit_job_partial_capacity(scheduler_commands, remote_command_executor)
-    _submit_job_full_capacity(scheduler_commands, remote_command_executor)
+    partial_capacity_log_assertions = scaling_behaviour_by_capacity["partial_capacity"]["log_assertions"]
+    full_capacity_log_assertions = scaling_behaviour_by_capacity["full_capacity"]["log_assertions"]
+
+    # Assert scaling behaviour when partial capacity for a job is available
+    submit_job_and_assert_logs(
+        scheduler_commands=scheduler_commands,
+        remote_command_executor=remote_command_executor,
+        job_kwargs=scaling_behaviour_by_capacity["partial_capacity"]["job"],
+        log_files=["/var/log/parallelcluster/slurm_resume.log"],
+        log_assertions=list(
+            partial_capacity_log_assertions["launch"]
```

Comment:
what about getting all values() from partial_capacity_log_assertions, without having to get them one by one?

Reply:
We can use `dict.values()`. We'll still need to flatten the list but at least we won't have to handle the log categories one by one.